Dockerfile

```diff
@@ -4,7 +4,8 @@ MAINTAINER Michael J. Stealey <stealey@renci.org>
 ### Begin - HydroShare Development Image Additions ###
 RUN pip install --upgrade pip && pip install \
   robot_detection \
-  django-ipware
+  django-ipware \
```

Comment:
@mjstealey It seems @aphelionz is waiting for an answer from you on the question above.